server/plugins/stencil-editor/stencil-editor.module.js

```diff
@@ -1,30 +1,30 @@
-var Handlebars = require('handlebars');
-var Hoek = require('hoek');
-var Path = require('path');
-var ThemeConfig = require('../../../lib/theme-config');
-var handlers = {};
-var internals = {
-    options: {}
+const Handlebars = require('handlebars');
+const Hoek = require('hoek');
+const Path = require('path');
+const ThemeConfig = require('../../../lib/theme-config');
+const handlers = {};
+const internals = {
+    options: {},
 };
-var themeConfig = ThemeConfig.getInstance();
+const themeConfig = ThemeConfig.getInstance();
 
-module.exports.register = function (server, options, next) {
+module.exports.register = (server, options, next) => {
     var configurationId;
```

Comment:
`let` is not supported in our current version of node without the `--harmony` flag